include/server/mir/graphics/display_configuration_observer.h

```diff
@@ -112,6 +112,15 @@ class DisplayConfigurationObserver
         std::shared_ptr<DisplayConfiguration const> const& failed_fallback,
         std::exception const& error) = 0;
 
+    /**
+     * Notification that the session should send the given display configuration
+     *
+     * It may or may not be a session configuration
+     */
+    virtual void session_should_send_display_configuration(
```

Comment:
I know this is a weird artefact of a transitional state in refactoring. But can we clarify the name slightly? I've not got a great name, but maybe `display_configuration_updated_for_this_session()`?